packages/redirect/src/class-redirect.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Jetpack Redirect package.
+ *
+ * @package  automattic/jetpack-redirect
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Class Redirect
+ */
+class Redirect {
+	/**
+	 * Constructor.
+	 *
+	 * Static-only class, so nothing here.
+	 */
+	private function __construct() {}
+
+	/**
+	 * Strip http:// or https:// from a url, replaces forward slash with ::,
+	 * so we can bring them directly to their site in calypso.
+	 *
+	 * @param string $url the full URL.
+	 * @return string $url without the guff
+	 */
+	private static function build_raw_urls( $url ) {
```

Comment:
What's the plan for the one in the Jetpack class?

Reply:
I didn't plan to touch it. it's still used in a few places